pkg/service/webhook/service.go

```diff
@@ -165,8 +166,17 @@ func (s Service) GetSupportedVerbs() GetSupportedVerbsResponse {
 	return GetSupportedVerbsResponse{Verbs: []Verb{Create, Delete}}
 }
 
-func (s Service) PublishWebhook(ctx context.Context, noun Noun, verb Verb, payloadReader io.Reader) {
-	webhook, err := s.storage.GetWebhook(context.Background(), string(noun), string(verb))
+func (s Service) PublishWebhook(noun Noun, verb Verb, payloadReader io.Reader) {
+	duration, err := time.ParseDuration(s.config.WebhookTimeout)
```

Comment:
nit: parse the duration when creating the service, instead of doing it for every call.